packages/create-jest/src/types.ts

```diff
@@ -17,7 +17,7 @@ export type PromptsResults = {
   useTypescript: boolean;
   clearMocks: boolean;
   coverage: boolean;
-  coverageProvider: boolean;
-  environment: boolean;
+  coverageProvider: string;
```

Comment:
Sorry for the extra diff, I had to fix `tslint` errors in this package. `jest-cli` code was ignored, but new package name ends with `jest` and it matches [the condition](https://github.com/dj-stormtrooper/jest/blob/feat/create-jest-package/scripts/lintTs.mjs#L56) in `tslint` script 😄

I decided to do a good thing and fix the errors instead of excluding the package from the script

,